interface-definitions/include/accel-ppp/radius-additions-rate-limit.xml.i

```diff
@@ -21,6 +21,20 @@
         <valueless />
       </properties>
     </leafNode>
+    <leafNode name="multiplier">
+      <properties>
+        <help>Shaper multiplier</help>
+        <valueHelp>
+          <format>&lt;0.001-1000&gt;</format>
+          <description>Shaper multiplier</description>
+        </valueHelp>
+        <constraint>
+          <regex>^[-+]?[0-9]*\.?[0-9]+$</regex>
```

Comment:
`^`  and `$` are no longer required - they are added implicitly to the regex when passed to the underlaying logic.

This regex also allows a value of 10000000000. Is it possible to use the range validator? Does it support floating point numbers? It would be the best fit!

Reply:
> 
> This regex also allows a value of 10000000000. Is it possible to use the numeric validator? Does it support floating point numbers? It would be the best fit!


@c-po  Numeric doesn't support "float"
I created a new validator `numeric-float`

Reply:
vyos/vyos-utils/pull/2 is now merged, so it's now trivial to validate floats.

Reply:
> [vyos/vyos-utils/pull/2](https://github.com/vyos/vyos-utils/pull/2) is now merged, so it's now trivial to validate floats.


@dmbaturin Thanks, updated PR